nntrainer/tensor/tensor_v2.h

```diff
@@ -382,8 +382,8 @@ class TensorV2 {
                 float value, float beta) noexcept;
 
   /**
-   * @brief 
-   * 
+   * @brief
```

Comment:
There is no brief.
Aren't you trying to include a brief about setZero?

Reply:
it was accidentally removed during the rebase. added back in #2439!